nl/guides/user-management/README.md

```diff
@@ -0,0 +1,107 @@
+
+Je kan de cursusbeheerders herkennen aan het icoontje in de linkermarge.
+
+![image](./staff.course_users_admin.png)
+
+Je kan de volgende actieknoppen aan de rechterkant gebruiken om cursusbeheerders aan te duiden:
+
+|knop                                                              |actie                                                                                |
+|------------------------------------------------------------------|-------------------------------------------------------------------------------------|
+![image](../../../images/staff_registration_icons/make_course_admin.png)    |cursusgebruiker zonder beheersrechten voor de cursus promoveren tot cursusbeheerder  |
+![image](../../../images/staff_registration_icons/make_student.png)         |cursusbeheerder degraderen tot cursusgebruiker zonder beheersrechten voor de cursus  |
+
+::: tip Belangrijk
+
+Een cursusbeheerder die zich uitschrijft uit een cursus verliest zijn status van cursusbeheerder.
+
+Om ervoor te zorgen dat een cursus altijd minstens één cursusbeheerder heeft, kan de laatste cursusbeheerder zich niet uitschrijven en kan hij zichzelf ook niet degraderen tot cursusgebruiker zonder beheersrechten voor de cursus.
+
+Na het aanmaken van een cursus kan een lesgever zich uitschrijven als hij andere cursusbeheerders aangeduid heeft. Hij kan door andere cursusbeheerders ook gedegradeerd worden tot cursusgebruiker zonder beheersrechten voor de cursus.
+:::
+
+### Registratiestatus aanpassen
+
+De cursusgebruikers worden in tabs gegroepeerd volgens hun registratiestatus voor de cursus. De volgende statussen zijn mogelijk:
+
+* `Geregistreerd`
+
+  Alle cursusgebruikers die momenteel geregistreerd zijn.
+
+* `Uitgeschreven`
+
+  Alle cursusgebruikers die ooit geregistreerd waren, maar die ondertussen uitgeschreven zijn.
+
+* `Op de wachtlijst`
+
+  Alle cursusgebruikers waarvoor er nog een registratieverzoek openstaat dat wacht op afhandeling door een cursusbeheerder.
+
+
+### Registratieverzoeken afhandelen
+
+Elke tab heeft eigen actieknoppen aan de rechterkant van de cursusgebruikers waarmee je hun registratiestatus kunt aanpassen.
+
+
+|tab                                                 |knop                    |actie |
+-----------------------------------------------------|------------------------|------| 
+`Geregistreerd`|          ![image](../../../images/staff_registration_icons/unregister.png) |  cursusgebruiker uitschrijven uit de cursus|
+`Uitgeschreven`|          ![image](../../../images/staff_registration_icons/register.png)|     cursusgebruiker terug registreren voor de cursus|
+`Registratieverzoeken`|   ![image](../../../images/staff_registration_icons/approve.png)|     goedkeuren van het registratieverzoek, waardoor de gebruiker geregistreerd wordt voor de cursus|
+`Registratieverzoeken`|   ![image](../../../images/staff_registration_icons/decline.png)|      afkeuren van het registratieverzoek, waardoor de gebruiker uitgeschreven wordt uit de cursus|
```

Comment:
This is very confusing because "tab" is first, it seems like the label for that icon which is not the case. I would remove the first column and start the description with "On the x tab, you can ..."